src/NUnitFramework/tests/Attributes/AuthorTests.cs

```diff
@@ -95,5 +95,33 @@ public void AuthorOnTestCase()
             var testCase = (Test)parameterizedMethodSuite.Tests[0];
             Assert.AreEqual("Charlie Poole", testCase.Properties.Get(PropertyNames.Author));
         }
+
+        #region Multiple Authors
+        [Test(Author = "Rob Prouse <rob@prouse.org>"),Author("Charlie Poole", "charlie@poole.org")]
+        [Author("NUnit")]
+        public void TestMethodMultipleAuthors()
+        {
+            Test test = TestBuilder.MakeTestFromMethod(FixtureType, "TestMethodMultipleAuthors");
+            var authors = test.Properties[PropertyNames.Author];
+            Assert.That(authors, Has.Exactly(3).Items);
+            Assert.AreEqual(authors[0], "Rob Prouse <rob@prouse.org>");
+            Assert.AreEqual(authors[1], "Charlie Poole <charlie@poole.org>");
+            Assert.AreEqual(authors[2], "NUnit");
```

Comment:
I am probably going to drop 
`Assert.That(authors, Has.Exactly(3).Items);`
as well.

And regarding TestCaseSource Author property, thoughts?